### DISCUSSION TITLE
More clarity in documentation

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#5366] [General]
In some parts of the documentation instructions are not quite clear what they are referring to.
For instance:

`Please ensure that any pivot attributes are listed in the withPivot() method of the relationship and inverse relationship.`

This phrase can be found in several parts of the documentation. But what it relates to or how to implement this is missing. If you try to find it in the documentation you are lost because this actually refers to a fundamental part of laravel itself and not filament. 

Suggestion: in such cases create a link to the documentation of laravel so the user is not lost.

**Comment by danharrin** [comment]
The docs are open to PR, it would be great if you could submit this!
